### PR TITLE
Migrate on deployment

### PR DESCRIPTION
**Pull request by emrojo**
Migrate database on start if SKIP_MIGRATIONS is defined as not true.

### CODE REVIEW

File: migrations/alembic.ini [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# A generic, single database configuration.
```
Comment (stevieing) [inline migrations/alembic.ini:1]: Might be worth adding some more explanation here as not sure why some of these attributes exist.

File: migrations/alembic.ini [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+keys = generic
+
+[logger_root]
+level = WARN
+handlers = console
+qualname =
+
+[logger_sqlalchemy]
+level = WARN
+handlers =
```
Comment (stevieing) [inline migrations/alembic.ini:29]: I can see that nothing is assigned here. Is there a reason for it being like this?

**Review by stevieing — APPROVED** [review]

File: migrations/alembic.ini (hunk 1 above)
Comment (emrojo) [reply migrations/alembic.ini:1]: This file was autogenerated. They are the logging configuration for when migrations are run.

Comment (emrojo) [reply migrations/alembic.ini:1]: Loggers represent the different components that could be logging while running the migration: sqlalchemy, alembic, flask_migrate or root (main app).
Each logger defines level of log (WARN, etc) and handlers (what to do with the message). The messages will go to the console as they are defined now.

**Pull request merged**